.dev/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# Root
```

Comment:
This file should be in project root right?

Reply:
Good catch. I believe it can be in the root or in `.github/`, but indeed not in `.dev/`